components/Kegiatan.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <div class="bg-kegiatan">
+    <div class="container my-6 mx-auto px-4 md:px-12">
+      <div class="text-kg text-center mb-8">
+        <br>
+        <br>
+        <h1>
+          Kegiatan
+        </h1>
+      </div>
+
+      <div class="flex flex-wrap -mx-1 lg:-mx-4">
+        <div class="my-1 px-1 w-full md:w-1/2 lg:my-4 lg:px-4 lg:w-1/3 mb-3">

```

Comment:
Kita akan menggunakan loop (v-for) untuk menampilkan sesuatu yang sama

Reply:
@xnuxer17